src/envoy/token/token_subscriber_test.cc

```diff
@@ -47,7 +49,8 @@ class TokenSubscriberTest : public testing::Test {
     mock_timer_ = new Envoy::Event::MockTimer{};
   }
 
-  void setUp(const TokenType& token_type) {
+  void setUp(const TokenType& token_type,
+             const DependencyErrorBehavior error_behavior) {
```

Comment:
Normally, we don't use `const T& t` for small data types, such as int, enum.   just  `T t`.  It is only used for large struct to avoid copying.

Reply:
Thanks, I read over go/totw/109 and understand now. Fixed